packages/tools/lib/generate-js-imports/illustrations.js

```diff
@@ -0,0 +1,73 @@
+const fs = require("fs").promises;
+const path = require('path');
+
+const generate = async () => {
+	const fioriInputFolder = path.normalize(process.argv[2]);
+	const tntInputFolder = path.normalize(process.argv[3]);
+	const outputFile = path.normalize(`${process.argv[4]}/Illustrations.js`);
+
+	const dirs = await fs.readdir(fioriInputFolder);
+	const fioriIllustrationsOnFileSystem = dirs.map(illustrationName => {
+		const fioriMatches = illustrationName.match(/.*\.js$/);
+		return fioriMatches ? illustrationName : undefined;
+	}).filter(key => !!key);
+
+	const tntDirs = await fs.readdir(tntInputFolder);
```

Comment:
minor things, "dir" not "dirs", "tntDir" not "tntDirs" as it's a single directory that we read